docs/pages/getting-started/bridges.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: What are bridges?
+description: Learn about different types of bridges and how they are used in blockchain networks.
+---
+
+
+
+# Bridges
+
+The web3 ecosystem is made up of a diverse array of blockchain networks, each with its own unique features, capabilities, and use cases. While this diversity is a key strength of the web3 ecosystem, it also presents challenges in terms of interoperability and communication between different networks. That is why we need bridges.
+
+## What are bridges?
+
+Blockchain bridges function similarly to physical bridges. Just as a physical bridge links two locations, a blockchain bridge connects two ecosystems.
+Bridges are fundamentally crucial to the suceess of web3, they give users the power to exchange information and transact freely. 
+
+By bridging networks, it enhances unrestricted flow of liquidity, capital efficiency, and accessibility within the decentralized finance (DeFi) landscape, while also promoting the scalability and sustainability of blockchain technologies. Additionally, they enable users to leverage the unique features and capabilities of different blockchains, creating a more inclusive and interconnected digital economy.
+
+{ /* explain on why we need bridges more */}
+{ /* explain the interoprability behind bridges */}
+
+
+## Traditional way of accessing multiple networks
+
+A centralized entity refers to an organization, institution, or system in which decision-making authority, control, or resources are concentrated in one central location or with one central authority. 
+
+A centralized entity in terms of exchanging crypto assets refers to a single organization or platform that controls and facilitates the buying, selling, and trading of cryptocurrencies on behalf of users. This entity typically operates as an intermediary, holding custody of users' funds and executing transactions through its platform. 
+
+Bridges provide a way to access multiple networks without having to go through centralized entities. This is important because centralized exchanges requires KYC that could take days depending on the amount of tokens you want to transfer, they are expensive and could be unreliable sometimes. They can also be hacked or shut down by regulators.
```

Comment:
Bridges provide a solution for interoperability among multiple networks.

Bridges can be designed with a central authority governing interactions between the interoperating networks or without one.

A centralized authority creates a single point of failure because it requires trusting that this entity will not be malicious and that it will not be co-opted by malicious actors, this is antithetical to the decentralized ethos of crypto.

Reply:
Thanks for the addition, I just included it into the docs.